spyderlib/plugins/configdialog.py

```diff
@@ -90,8 +91,21 @@ def apply_changes(self):
             self.save_to_conf()
             if self.apply_callback is not None:
                 self.apply_callback()
+
+            # Since the language cannot be retrieved by CONF and the language
+            # is needed before loading CONF, this is an extra method needed to
+            # ensure that when changes are applied, they are copied to a
+            # specific file storing the language value. This only applies to
+            # the main section config.
+            if self.CONF_SECTION == u'main':

```

Comment:
`main` is the section in charge of setting the interface language, so the `_save_lang` only belongs to that section, before I was using  `if getattr(self, "_save_lang", False):` but I change it so that it was more explicit.

Since this method runs for every child (any other section or plugin) then I need this to avoid an error of calling _save_lang when it does not exist.